### PR TITLE
CURA-3634: Make prime blob optional

### PR DESCRIPTION
**Pull request by LipuFei**

### CODE REVIEW

File: resources/definitions/fdmprinter.def.json [hunk 1]
```diff
@@ -3290,6 +3290,16 @@
             "description": "Adhesion",
             "children":
             {
+                "prime_enable":
+                {
+                    "label": "Enable Prime",
```
Comment (BagelOrb) [inline resources/definitions/fdmprinter.def.json]: I'm not sure whether this naming is good.
Priming is making the printer ready. This involves unretracting as well.

Did you have a talk with somebody about the best name for this setting?

Maybe Menno is the right guy?
Also Jeroen from support might have something to say about this.

File: resources/definitions/fdmprinter.def.json [hunk 2]
```diff
@@ -3290,6 +3290,16 @@
...
+                    "description": "Whether to prime the filament before printing.",
```
Comment (BagelOrb) [inline resources/definitions/fdmprinter.def.json]: Please explain what benefit the user has by changing this setting.

File: resources/definitions/fdmprinter.def.json [hunk 3]
```diff
@@ -3290,6 +3290,16 @@
             "description": "Adhesion",
             "children":
             {
+                "prime_poop_enable":
+                {
+                    "label": "Enable Prime Blob",
+                    "description": "Whether to prime the filament with a blob before printing.",
```
Comment (BagelOrb) [inline resources/definitions/fdmprinter.def.json]: Please explain what benefit the user has by changing this setting.

The description should explain to a user why and when he should change the setting.

It should explain the benefit of turning it off and the benefit of turning it off.

**Comment by LipuFei** [comment]
@BagelOrb updated the setting name and the description.

**Pull request merged**